main.go

```diff
@@ -282,6 +287,24 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 			} else {
 				s.ChannelMessageSend(m.ChannelID, rps(s, m, rpsChoice))
 			}
+		case botPrefix + "nickchange":
+			st, err := s.UserChannelCreate(m.Author.ID)
+			if err != nil {
+				s.ChannelMessageSend(m.ChannelID, fmt.Sprintf("could not create a private channel for %v", m.Author))
+			}
+			nickArgs := strings.SplitN(m.Content, " ", 4)
+			user := nickArgs[1]
```

Comment:
This causes an error when the command is executed as:
`!changenick @_user1234 newnickname`

But works when command is executed as:
`!changenick 35283643374619853 newnickname`
(where the number is a user's ID)

Could you make it such that it works both ways? To do this, check for the presence of the character `<` in the user string. If it is present, trim the characters `<`, `>` and `@` from the user string. This will ensure that the command works in all cases.